### DISCUSSION TITLE
Receiver - Filelog - Labels based on the path (or filename) ? Or solution with processor ?

### DISCUSSION THREAD

**Discussion by DamienCH** [open-telemetry/opentelemetry-collector-contrib#31697] [Q&A]
Hello,

This probably isn't a very clever question, but I can't figure it out. I'm new to anything logging and am at a loss now.

Context:
I have built a collector which will be picking logs in a fileshare from multiple apps/services/servers. Then the collector would send the logs to Loki.

My problem:
I would like to set specific labels, ideally the label would be based on the folder path.  Is it possible and how could I accomplish that ?

If these are things I can/should accomplish with processors instead, could anyone give me a guidance or an example ?

I've spent way too much time trying to figure that and other stuff, any tab leads me to 6 other tabs and more questions.
Some help would be greatly appreciated.

Thanks !

Edit: Removing my original question about the wildcard in the path. I just retried and got it to work, not sure how I messed that earlier.

**Comment by Frapschen** [comment]
Are you use [filelogreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver) to ingest your log files? if so, the exporter has `include_file_path ` attr, you can use [transformprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor) to generate the specific labels whatever you want
